packages/shared/src/components/CreateMyFeedButton.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { ReactElement, useContext, useEffect } from 'react';
+import classNames from 'classnames';
+import { IFlags } from 'flagsmith';
+import PlusIcon from '../../icons/plus.svg';
+import UserIcon from '../../icons/user.svg';
+import { Button } from './buttons/Button';
+import { Features, getFeatureValue } from '../lib/featureManagement';
+import AnalyticsContext from '../contexts/AnalyticsContext';
+import { AnalyticsEvent } from '../hooks/analytics/useAnalyticsQueue';
+import { getThemeColor, ThemeColor } from './utilities';
+
+interface CreateMyFeedButtonProps {
+  type: string;
+  sidebarExpanded?: boolean;
+  flags: IFlags;
+  action: () => unknown;
+}
+
+const getAnalyticsEvent = (
+  eventName: string,
+  copy: string,
+  type: string,
+): Partial<AnalyticsEvent> => ({
+  event_name: eventName,
+  target_type: 'my feed button',
+  target_id: type,
+  feed_item_title: copy,
+});
+
+interface ClassProps {
+  explainerColor?: ThemeColor;
+  sidebarExpanded?: boolean;
+}
+const wrapperClasses = (sidebarExpanded: ClassProps) => {
+  return {
+    sidebar: sidebarExpanded ? 'h-[8.125rem] justify-center' : 'h-[8.125rem] ',
```

Comment:
Both of them have the same height, I think we can do the condition on the centering part only.
```suggestion
    sidebar: classNames('h-[8.125rem]', sidebarExpanded && 'justify-center'),
```